apps/src/shareWarnings.js

```diff
@@ -52,7 +52,7 @@ function handleShareWarningsTooYoung(onTooYoung) {
   utils.trySetLocalStorage('is13Plus', 'false');
   if (onTooYoung) {
     onTooYoung();
-  } else {
+  } else if (!IN_UNIT_TEST) {
```

Comment:
it won't end up in production code. webpack will spit this out as `} else if (false) {` and then the minifier will strip the entire if block.